notebooks/entities-synonyms/theasaurus.py

```diff
@@ -38,3 +47,12 @@
               indent=4)
 
 del csv_file, entities_single, file, tic
+
+# format to dataframe for knowledge graph
+df_synonyms = pd.DataFrame.from_dict(data=entities_single_synonyms, orient='index')
+df_synonyms = df_synonyms.reset_index()
+df_synonyms = df_synonyms.melt(id_vars='index', value_name='synonym')
+df_synonyms = df_synonyms.dropna(subset=['synonym'])
+df_synonyms = df_synonyms.rename(columns={'index': 'entity'})
+df_synonyms[['entity', 'synonym']].to_csv(path_or_buf='data/processed/api_synonyms.csv',
```

Comment:
As above wrt ensuring uniqueness

Reply:
Done in https://github.com/alphagov/govuk-entity-personalisation/pull/5/commits/e11cec671aa29a3097db56ed3856275cb128644b